Jenkinsfile

```diff
@@ -8,9 +8,11 @@ pipeline {
     stage('Test') {
       steps {
         sh '''rm -rf build/reports
+        export LANG=en_US.UTF-8
+        export LANGUAGE=en_US.UTF-8
+        export LC_ALL=en_US.UTF-8
```

Comment:
this fixes an issue where some output (including new diagnostic output from the localization script as a result of this pr) would cause fastlane to crash while running the build